precice.pyx

```diff
@@ -139,16 +430,47 @@ cdef class Interface:
         self.thisptr.setMeshVertices (mesh_id, size, <const double*>_positions.data, <int*>_ids.data)
         return _ids
 
-    # get vertex positions for multiple vertex ids from a given mesh
     def get_mesh_vertices(self, mesh_id, ids):
+        """
+        Get vertex positions for multiple vertex ids from a given mesh
+
+        Parameters:
+            mesh_id (int): ID of the mesh to read the vertices from.
+            ids (array): The ids of the vertices to lookup
```

Comment:
`numpy.array` of shape `N`

Reply:
This change was also implemented in other locations where it was relevant to mention that not just an `array` but a `numpy.array` of shape `N` is used